docs/flyway.md

```
@@ -0,0 +1,29 @@
+# Flyway Migration Guide
+
+## 1. 경로 고정
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)